pkg/kube/cluster-init.sh

```diff
@@ -3,17 +3,22 @@
 # Copyright (c) 2023 Zededa, Inc.
 # SPDX-License-Identifier: Apache-2.0
 
-K3S_VERSION=v1.26.3+k3s1
+K3S_VERSION=v1.28.5+k3s1
 KUBEVIRT_VERSION=v0.59.0
 LONGHORN_VERSION=v1.6.0
 CDI_VERSION=v1.57.0
+EVE_EXTERNAL_BOOT_IMG_TAG=325e7463e093db2c1abc0da0260b193e73bcef5d
+EVE_EXTERNAL_BOOT_IMG=docker.io/lfedge/eve-external-boot-image:${EVE_EXTERNAL_BOOT_IMG_TAG}
 NODE_IP=""
 MAX_K3S_RESTARTS=10
 RESTART_COUNT=0
-
-INSTALL_LOG=/var/lib/install.log
-CTRD_LOG=/var/lib/containerd-user.log
+K3S_LOG_DIR="/persist/newlog/kube"
```

Comment:
Where do we manage space for this directory?

Reply:
@eriknordmark Lines 527-531 with check_log_file_size()